config/locales/views/articles/en.yml

```diff
@@ -48,6 +48,10 @@ en:
         aria_label: Save to reading list
         initial: Save
         success: Saved
+      scheduled:
+        subtitle: Scheduled Post.
+        text: " This URL is public but secret, so share at your own discretion."
```

Comment:
Reading note: The leading space in `" This URL...` is intentional, see the fr.yml file for details.